main/opcodes.zkasm

```diff
@@ -2330,13 +2329,12 @@ opSaveTopicsLoop:
 ; // TODO Check with DElEGATE CALL txDestAddr/txSrcAddr ( same as other calls)
 ; // TODO check revert create
 opCREATE:
-
     SP - 3              :JMPN(stackUnderflow)
     $ => A              :MLOAD(isStaticCall)
     0 - A               :JMPN(invalidStaticTx)
     SP - 1 => SP
-    $ => A              :MLOAD(SP--)            ;value
-    A                   :MSTORE(valueCall)
+    $ => D              :MLOAD(SP--)            ;value
```

Comment:
Why changing register to D?

Reply:
bc i can sue it later without to loading it again ^^